tensorflow/compiler/xla/service/gpu/tests/hlo_to_llvm_ir.cc

```diff
@@ -101,7 +106,7 @@ xla::Status CompileAndPrintLlvmIr(const std::string& hlo_text,
                             llvm_module.get(), cuda_compute_capability,
                             hlo_module->config(), libdevice_dir));
     std::cout << ptx << std::endl;
-#else
+#elif TENSORFLOW_USE_ROCM
     std::string libdevice_dir = tsl::RocdlRoot();
     xla::gpu::GpuVersion gpu_version{rocm_compute_capability};
     TF_ASSIGN_OR_RETURN(
```

Comment:
ptx is nvidia thing, I think maybe we can change to std::vector<uint8_t> hasco_

Reply:
If you want, you can follow this PR up with one that cleans up the language and variable names in this file.

Reply:
Yes, I will keep following up.